src/Pages/ComparisonGraph.cshtml

```diff
@@ -0,0 +1,50 @@
+﻿@page
+@using System.Globalization
+@model ePiggyWeb.Pages.ComparisonGraphModel
+
+<form method="get" class="mt-5">
+    <input type="hidden" name="handler" value="Filter" />
+    <input for="startDate" asp-for="StartDate" type="date" value="@Model.StartDate.ToShortDateString()" />
+    <input for="endDate" asp-for="EndDate" type="date" value="@Model.EndDate.ToShortDateString()" />
+    <input type="submit" value="Filter" class="btn btn-warning ml-5" />
+</form>
+
+<div class="container p-0 mt-3 mb-5">
+    <div class="col-12" id="graph"> <div class="spinner-border text-warning loader" role="status">
+        <span class="sr-only">Loading...</span>
+    </div></div>
+</div>
+
+
+
+
+<script type="text/javascript" src="https://www.gstatic.com/charts/loader.js"></script>
+<script type="text/javascript">
+    // Load google charts
+    google.charts.load('current', { 'packages': ['corechart'] });
+    google.charts.setOnLoadCallback(drawChart);
+
+    // Draw the chart and set the chart values
+    function drawChart() {
+        var data = google.visualization.arrayToDataTable([
+            ['Type', 'Amount'],
+            ['Expenses', @Model.Expenses.ToString(CultureInfo.InvariantCulture)],
+            ['Income',  @Model.Income.ToString(CultureInfo.InvariantCulture)]
```

Comment:
Could you explain why you're using invariant culture here?

Reply:
when I use plain decimal or smth it gives me number with comma and the graph won't render it, so only solution I found was to convert to string with invariant culture.